src/main/java/uk/gov/hmcts/juror/job/execution/jobs/checks/morning/DashboardMorningChecksJob.java

```diff
@@ -177,28 +173,20 @@ public void addTableRow(String title, String message, TableItem.Status status) {
             this.tableItems.add(new TableItem(title, message, status));
         }
 
-        public void saveToFileAndEmail(Result result) {
+        public void buildHtmlAndSaveToFile(Result result) {
             try {
                 String htmlResponse = buildHtml();
                 saveToFile(htmlResponse);
-                email(result, htmlResponse);
             } catch (Exception e) {
-                log.error("Failed to save and email dashboard morning checks", e);
-                throw new InternalServerException("Failed to save and email dashboard morning checks", e);
+                log.error("Failed to save dashboard morning checks", e);
+                throw new InternalServerException("Failed to save dashboard morning checks", e);
             }
         }
```

Comment:
We probs dont need this step anymore as there is no way to view the file.

We need to depend on the stats from the schedular meta data

Reply:
Until confirmation is received the data is available via Dashboard, this step can remain.

Reply:
I don't think this will be available via dashboard?

Reply:
Method removed.